Task1/script.js

```diff
@@ -0,0 +1,51 @@
+const input = document.getElementById('rpn');
+const output = document.getElementById('result');
+
+async function add(firstNumber, secondNumber){
+  await new Promise(r => setTimeout(r, 2000));
+  return firstNumber + secondNumber;
+}
+
+async function subtract(firstNumber, secondNumber){
+  await new Promise(r => setTimeout(r, 1000));
+  return firstNumber - secondNumber;
+}
+
+async function multiply(firstNumber, secondNumber){
+  await new Promise(r => setTimeout(r, 3000));
+  return firstNumber * secondNumber;
+}
+
+async function divide(firstNumber, secondNumber){
+  await new Promise(r => setTimeout(r, 4000));
+  return firstNumber / secondNumber;
```

Comment:
why don't you return the promise right away? `return new Promise(r => setTimeout(() => {r(firstNumber / secondNumber)}, 4000));`